packages/auth/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@recidiviz/auth",
+  "version": "1.0.0",
+  "description": "Auth0-Powered Authentication Module",
+  "author": "Recidiviz <team@recidiviz.org>",
+  "license": "GPL-3.0-only",
+  "engines": {
+    "node": ">=12.x.x"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c",
+    "analyze": "rollup -c --environment ANALYZE",
+    "watch": "run-p watch:build watch:sync",
+    "watch:build": "rollup -c -w",
+    "watch:sync": "nodemon --watch dist --exec 'yalc push'",
+    "lint": "tsc && eslint --ext .js,.ts,.tsx src/",
+    "test": "react-scripts test"
+  },
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^1.22.1",
+    "mobx": "^6.6.1",
+    "mobx-react-lite": "^3.4.0",
+    "qs": "^6.11.0",
+    "@testing-library/dom": "^8.11.3",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.0.0",
+    "react-scripts": "5.0.0"
```

Comment:
I think these should all be devDependencies or else they'll get installed downstream

Reply:
Ah - thanks, Ian!